app/src/main/java/com/example/dine_and_donate/MapActivity.java

```diff
@@ -106,6 +107,7 @@
 
     private View slideView;
     private boolean slideViewIsUp;
+    Button btnCreate;
```

Comment:
this can be private and you should prefix it with "m" (same with the other variables, if you want to fix them)